plugins/button/plugin.js

```diff
@@ -308,7 +310,8 @@
 				focusFn: focusFn,
 				clickFn: clickFn,
 				style: CKEDITOR.skin.getIconStyle( iconPath, ( editor.lang.dir == 'rtl' ), overridePath, this.iconOffset ),
-				arrowHtml: this.hasArrow ? btnArrowTpl.output() : ''
+				arrowHtml: this.hasArrow ? btnArrowTpl.output() : '',
+				toggleHtml: this.isToggle ? 'aria-pressed="false"' : ''
```

Comment:
Since it is part responsible only for arai attribute, then maybe:
```suggestion
				toggleAriaHtml: this.isToggle ? 'aria-pressed="false"' : ''
```